lib/internal/cluster/master.js

```diff
@@ -41,15 +42,15 @@ if (schedulingPolicy === undefined) {
 
 cluster.schedulingPolicy = schedulingPolicy;
 
-cluster.setupMaster = function(options) {
+cluster.setupMaster = function(options = {}) {
```

Comment:
This is a silly one, but this might be a breaking change, but IMHO we should keep it unless someone objects.
This changes the `.length` of the method:
```js
> require('cluster').setupMaster.length
1
> (function a(a1) {}).length
1
> (function b(b1={}) {}).length
0
```